deploy/lingo_deploy.yaml

```diff
@@ -24,3 +24,14 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 8081
+            initialDelaySeconds: 1
+            periodSeconds: 5
+          livenessProbe:
+            tcpSocket:
```

Comment:
Tcp should be fine for liveness but if preferred, the health endpoint be configured to serve a liveness response, too
Again, not sure about the default time values here